package dawid.kotarba.shared.model.exceptions

/**
  * Created by Dawid on 12.07.2016.
  */
object ExceptionType extends Enumeration {
  val NOT_FOUND, INTERNAL_ERROR = Value
}
